public/app/plugins/datasource/loki/LogContextProvider.ts

```diff
@@ -363,8 +373,10 @@ export class LogContextProvider {
         // If we end up with no real labels enabled, we need to reset the init filters
         return contextFilters;
       } else {
-        // Otherwise use new filters
-        if (arePreservedLabelsUsed) {
+        // Otherwise use new filters; also only show the notification if filters
+        // are supposed to be cached, which is currently used in the UI, not
+        // when tab-opened
+        if (arePreservedLabelsUsed && cacheFilters) {
```

Comment:
I think that instead of this, we should not get to this logic and keep `preservedLabels` as false, as we are not using them. I don't think we even need to check local storage for preserved labels in that case. So we could do, even before checking for `preservedLabels`:
```
if (!cacheFilters) {
return contextFilters;
}
```

Also I noticed that we are not passing `cacheFilters` in LokiContextUi.

Reply:
No, we are using them, it's just that we don't show the popup that filters are applied when the context is opened in a new tab.

Reply:
> Also I noticed that we are not passing cacheFilters in LokiContextUi.

That's fine, as we always want to cache and the default is set to `true`.